docs/developer-guide/serialization.rst

```diff
@@ -20,6 +22,148 @@ For more information on the formats that KSQL supports, see :ref:`ksql_formats`.
 
 KSQL provides some additional configuration that allows serialization to be controlled:
 
+.. _ksql_single_field_wrapping
+
+-------------------------
+Single field (un)wrapping
+-------------------------
+
+.. note:: The ``DELIMITED`` format is not effected by single field unwrapping.
+
+Controlling deserializing of single fields
+==========================================
+
+When KSQL deserializes a Kafka message into a row, the key is deserialized into
+the key field and the message's value is deserialized into the value field(s).
+
+By default, KSQL expects any value with a single-field schema to have been serialized as a named
+field within a record. However, this is not always the case. KSQL also supports reading data
+that has been serialized as an anonymous value.
+
+For example, a value with multiple fields might look like the following in JSON:
+
+.. code:: json
+
+   {
+      "id": 134,
+      "name": "John"
+   }
+
+If the value only had the ``id`` field, KSQL would still expect the value to be serialized as a
+named field, for example:
+
+.. code:: json
+
+   {
+      "id": 134
+   }
+
+If your data contains only a single field, and that field is not wrapped within a JSON object,
+(or an Avro record if using the ``AVRO`` format), then you can use the ``WRAP_SINGLE_VALUE``
+property in the ``WITH`` clause of your :ref:`CREATE TABLE <create-table>` or
+:ref:`CREATE STREAM <create-stream>` statements. Setting the property to ``false`` will inform KSQL
+that the value is not wrapped, i.e. the example above would be a JSON number:
+
+.. code:: json
+
+     134
+
+For example, the following creates a table where the values in the underlying
+topic have been serialized as an anonymous JSON number:
+
+.. code:: sql
+
+    CREATE TABLE x (ID INT) WITH (WRAP_SINGLE_VALUE=false, ...);
```

Comment:
Maybe a slightly more descriptive name than `x`. :)

Reply:
What would you prefer? 'Bob'?

Reply:
Heh, I dunno, maybe something like `table-no-wrap-single-value` ... I'm all about verbose, self-describing object names, but ymmv.